dockerfiles/base/scripts/base/commands/cmd_action.sh

```diff
@@ -10,9 +10,31 @@
 #
 
 cmd_action() {
-  debug $FUNCNAME
+  docker_run -it ${UTILITY_IMAGE_CHEACTION} "$@"
+}
+
+pre_cmd_action() {
```

Comment:
if we call utility image che action with --help it displays the help automatically so we shouldn't write again the help there as it is duplicating the help (and then we will have errors of data being not consistent)
we should only delegate